docs/website/src/main/resources/certifications/jakarta-authentication/3.0/TCK-Results.md

```diff
@@ -0,0 +1,193 @@
+TCK Results
+===========
+
+As required by the
+[Eclipse Foundation Technology Compatibility Kit License](https://www.eclipse.org/legal/tck.php),
+following is a summary of the TCK results for releases of Jakarta Authentication.
+
+# Eclipse GlassFish 7.0 Certification Summary
+
+- Product Name, Version and download URL (if applicable): <br/>
+  [Eclipse Glassfish 7.0.0-M4](https://github.com/eclipse-ee4j/glassfish/releases/download/7.0.0-M4/glassfish-7.0.0-M4.zip)
+- Specification Name, Version and download URL: <br/>
+  [Jakarta Authentication 3.0](https://jakarta.ee/specifications/authentication/3.0/)
+- TCK Version, digital SHA-256 fingerprint and download URL: <br/>
+  [Jakarta Authentication TCK, 3.0.0](https://download.eclipse.org/es/jakartaee10/staged/eftl/jakarta-authentication-tck-3.0.0-dist.zip), SHA-256: `f970fce5dd3883934ef4f1162c747c527f98cf88fcacc8d7af6ae4cf444c9ef8`
```

Comment:
I'd expect TCK to be downloaded from `/ee4j/jakartaee10/...`.
Perhaps I missed something that justifies `/es`.

Reply:
> I'd expect TCK to be downloaded from /ee4j/jakartaee10/....
> Perhaps I missed something that justifies /es.

It's the only location writable for the readily available security related CI. The other locations aren't writable, issues have been filed, but today is the deadline. We'll have to move it later and update this.

Reply:
I checked with @ivargrimstad and it appears that `/es` is absolutely fine. However, by then I had already moved it to `/ee4j/jakartaee10/...` and re-run the TCKs.